NOnion/Utility/HiddenServicesUtility.fs

```diff
@@ -6,9 +6,10 @@ open NOnion
 open DateTimeUtils
 
 module HiddenServicesUtility =
-    let GetTimePeriod (now: DateTime) (hsDirInterval: int) =
+    let GetTimePeriod (lastConsensusValidAfter: DateTime) (hsDirInterval: int) =
```

Comment:
I said the rename should go in a different commit, man

Reply:
Yeah, I know, fixed it as soon as I noticed.